jetty-http-spi/src/main/java/org/eclipse/jetty/http/spi/DelegatingThreadPool.java

```diff
@@ -63,16 +63,16 @@ public boolean tryExecute(Runnable task)
     }
 
     @Override
-    public boolean isUseVirtualThreads()
```

Comment:
Do we have a release with this name?
If so, this can't be removed, only deprecated.